lib/ace/autocomplete.js

```diff
@@ -143,7 +143,7 @@ var Autocomplete = function() {
         // on IE preventDefault doesn't stop scrollbar from being focussed
         var el = document.activeElement;
         var text = this.editor.textInput.getElement()
-        if (el != text && el.parentNode != this.popup.container
+        if (el != text && this.popup && el.parentNode != this.popup.container
```

Comment:
when popup is null should this ignore blur event or call detach to remove event listneners?